src/components/Text/Text.js

```diff
@@ -8,38 +8,27 @@ import { sizes } from '../../styles/constants';
 
 const { KILO, MEGA, GIGA } = sizes;
 
+const responsiveMap = {
```

Comment:
Hm, I wonder if we can make the purpose of this more obvious. Maybe call it `mobileSizeMap` or add a component?